### ISSUE TITLE
Unnecessary (read-data) after save-text?

### ISSUE THREAD

**Issue by ghollisjr** [herth/ltk#19]
While merging in this repo's master to mine, I noticed that save-text from ltk/ltk.lisp line 3416 has an extra (read-data) at the end of the function that I removed while fixing a bug in the past issues (I'll have to dig into the logs to figure out which one).  Is there a reason to call read-data after a save-text operation?

**Comment by herth** [comment]
Yes, because save-text sends some data to lisp, which needs to be read to keep in sync. Of course, one could conceive save-text to be asynchronous and remove the handshake entirely, but I think it is better that the user knows that the operation succeeded if the function returns.
